gadget/install/params.go

```diff
@@ -41,3 +43,31 @@ type InstalledSystemSideData struct {
 	// is the raw encrypted device node (eg. /dev/mmcblk0p1).
 	DeviceForRole map[string]string
 }
+
+// partEncryptionData contains meta-data for an encrypted partition.
+type partEncryptionData struct {
+	role            string
+	device          string
+	encryptedDevice string
+
+	volName             string
+	encryptionKey       keys.EncryptionKey
+	encryptedSectorSize quantity.Size
```

Comment:
so this is not really needed, unless we decide to pass it back to the installer but then for an installer it probably make more sense to find out with its own code what it needs to know in term of sizes and sector sizes